bundles/org.openhab.binding.telegram/src/main/java/org/openhab/binding/telegram/internal/action/TelegramActions.java

```diff
@@ -108,34 +108,30 @@ public String toString() {
 
     @RuleAction(label = "send an answer", description = "Send a Telegram answer using the Telegram API.")
     public boolean sendTelegramAnswer(@ActionInput(name = "chatId") @Nullable Long chatId,
-            @ActionInput(name = "replyId") @Nullable String replyId,
+            @ActionInput(name = "callbackId") @Nullable String callbackId,
+            @ActionInput(name = "messageId") @Nullable Long messageId,
```

Comment:
Does this create a breaking change?

Reply:
@CrazyIvan359 Can you answer this question?
It seems to be breaking the existing action. So if this is the case, please create a PR similar to https://github.com/openhab/openhab-distro/pull/1314/files for the distro.